### ISSUE TITLE
[terra-application-links] Themeable variable issues

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-framework#1170]
# Bug Report

The following variables are duplicated:

--terra-application-links-tabs-tab-transition-timing-function - packages/terra-application-links/lib/tabs/clinical-lowlight-theme/ApplicationTabs.module.scss (line: 31, column: 4)
--terra-application-links-tabs-tab-transition-timing-function - packages/terra-application-links/lib/tabs/clinical-lowlight-theme/ApplicationTabs.module.scss (line: 59, column: 4)

--terra-application-links-tabs-tab-transition-timing-function - packages/terra-application-links/lib/tabs/orion-fusion-theme/ApplicationTabs.module.scss (line: 31, column: 4)
--terra-application-links-tabs-tab-transition-timing-function - packages/terra-application-links/lib/tabs/orion-fusion-theme/ApplicationTabs.module.scss (line: 59, column: 4)

The following variables are missing:

**clinical-lowlight-theme**

--terra-application-links-demo-header-background-color - packages/terra-application-links/lib/terra-dev-site/doc/example/demoStyles.module.scss (line: 3, column: 4)
--terra-application-links-tabs-container-padding-right - packages/terra-application-links/lib/tabs/ApplicationTabs.module.scss (line: 27, column: 4)

**orion-fusion-theme**

--terra-application-links-demo-header-background-color - packages/terra-application-links/lib/terra-dev-site/doc/example/demoStyles.module.scss (line: 3, column: 4)
--terra-application-links-tabs-container-padding-right - packages/terra-application-links/lib/tabs/ApplicationTabs.module.scss (line: 27, column: 4)
